### ISSUE TITLE
.addTags not a member

### ISSUE THREAD

**Issue by ethanfox** [ElaWorkshop/TagListView#114]
Whenever trying to add multiple tags, Xcode gives an error that addTags is not a member of TagListView.

**Comment by Cee** [comment]
Have you created an instance of `TagListView`?

For example:
```swift
let tagListView = TagListView()
tagListView.addTags(["Add", "three", "tags"])
// and so on
```

**Comment by ahbou** [comment]
I was having the same issue until I pointed the podfile to this repo

`pod 'TagListView', :git => 'https://github.com/ElaWorkshop/TagListView'`

**Comment by Cee** [comment]
I'll create a new tag for this repo.

**Comment by Cee** [comment]
@ahbou @entotsu Have you tried the latest version of `TagListView`? This bug should be fixed along with issue #116 .
